docs/reference/eql/search.asciidoc

```diff
@@ -213,3 +213,33 @@ GET /sec_logs/_eql/search
 }
 ----
 ====
+
+[discrete]
+[[eql-search-case-sensitive]]
+=== Run a case-sensitive EQL search
+
+By default, matching for EQL queries is case-insensitive. You can use the EQL
+search API's `case_sensitive` parameter to toggle case sensitivity on or off.
+
+.*Example*
+[%collapsible]
+====
+The following request searches contains a query that matches `process` events
```

Comment:
`The following search request contains a query....` maybe? I am not sure I understand "The following request searches contains a query..."

Reply:
Thanks for catching this. That's a typo on my part. Your suggestion is correct.